common/src/test/scala/uk/ac/wellcome/test/utils/AmazonCloudWatchLocal.scala

```diff
@@ -0,0 +1,13 @@
+package uk.ac.wellcome.test.utils
+
+import com.amazonaws.client.builder.AwsClientBuilder.EndpointConfiguration
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchClientBuilder
+
+trait AmazonCloudWatchLocal {
+  val amazonCloudWatch = AmazonCloudWatchClientBuilder
+    .standard()
+    .withEndpointConfiguration(
+      // use a fake endpoind in tests so that we don't send metrics to the real AWS
+      new EndpointConfiguration("http://localhost:6789", "local"))
```

Comment:
None. There's no service listening to it, it's just t drown the metrics somewhere when running tests